workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -222,6 +253,13 @@ def compute_percentile(
     return bin_midpoints[closest_index]
 
 
+def itcz_edges(psi: xr.DataArray, lat: str = "latitude",) -> Tuple[float, float]:
+    """Compute latitude of ITCZ edges given mass streamfunction at particular level."""
+    lat_min = psi.sel({lat: slice(-30, 10)}).idxmin(lat).item()
+    lat_max = psi.sel({lat: slice(-10, 30)}).idxmax(lat).item()
```

Comment:
sometimes people will fit a polynomial near the region of the min/max in order to identify the min/max positions on a scale finer than the grid resolution (in this case 2°).